src/map/latent_effect.h

```diff
@@ -71,7 +71,7 @@ enum LATENT
     LATENT_JOB_LEVEL_ODD            = 41,
     LATENT_JOB_LEVEL_EVEN           = 42,
     LATENT_WEAPON_DRAWN_HP_UNDER    = 43, //PARAM: HP PERCENT
-    LATENT_STATUS_ACTIVE_NO_JOBSPECIAL = 44, //PARAM: EFFECTID - When status effect active, but without a 1hr/2hr Job Special active
+    LATENT_STATUS_ACTIVE_NO_JOBSP   = 44, //PARAM: EFFECTID - When status effect active, but without a 1hr/2hr Job Special active
```

Comment:
since this is currently only applicable on 100 fists, can we add that information to the definitions here and in status lua? otherwise people may try to use it to no effect.

Reply:
Should we just add an effect flag to check against? I thought we had a way to tell if something was a 2hr already but I've been unable to locate it. We could nip it in the bud right now.

Reply:
I think I agree with these guys; just making the all-SP change now will stop the TODO sitting there and rotting forever, especially because it isn't much extra work.

Reply:
@Era-Lusiphur example of using an effectglag:
https://github.com/project-topaz/topaz/commit/d099abc12c8f682520672f7c6acb7435f12857e3

adding the ffect flag used above:
https://github.com/project-topaz/topaz/commit/53398a83daf8734319e98800723a1d60a61d88a0

So we could flag 2hr statuses, and then check the flag. alternatively you can check each status, or come up with a 3rd option - effectflag is just all I could think of to avoid a bunch of `||` to check multiple effects.